### DISCUSSION TITLE
MyST-Parser docutils support

### DISCUSSION THREAD

**Discussion by chrisjsewell** [executablebooks/MyST-Parser#487] [General]
This topic is intended for any discussion relating to myst-parser's support of docutil-only use.
See also: https://sourceforge.net/p/docutils/feature-requests/86/

**Comment by chrisjsewell** [comment]
See also #347, for some previous discussion

**Reply by memeplex** [reply]
As someone new to myst-parser who was looking for a replacement to pandoc that just knows how to dump html and latex from markdown, this was my experience:
1. I installed myst-parser as documented and realized that a bunch of sphinx dependencies were included.
2. I googled for a way of installing myst-parser without sphinx and somewhat randomly found https://pypi.org/project/myst-docutils. Since the source was executablebooks I felt confident enough to install it.
3. I had to set `PYTHONWARNINGS=ignore` in the CLI because of many `FutureWarnings` thrown by myst-docutils-xxx.

So a couple of things to improve:
- Please document somewhere that myst-docutils is an option to myst-parser. Preferently in the Get Started page.
- Avoid throwing warnings intended to programmers from user facing commands.

My 2c.

**Reply by chrisjsewell** [reply]
> Please document somewhere that myst-docutils is an option to myst-parser.

Heya, just to note https://myst-parser.readthedocs.io/en/latest/docutils.html

**Reply by memeplex** [reply]
I've read that page at least twice. What I meant is that the existence of the `myst-docutils` package itself is not documented AFAICS. From reading that page I can only conclude that it's possible to *use* myst without sphinx, but not that it is possible to *install* it without sphinx. That's why I suggested to put this bit of information in the Get Started page, next to the standard installation instructions, something like "If you don't want to install sphinx, you can install myst-docutils instead of myst-parser".
